docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -83,14 +86,48 @@ Lets examine the query:
 The result of this query would be a list of User objects where all
 users are older than 20.
 
-The SELECT clause allows to specify both class identification
-variables that signal the hydration of a complete entity class or
-just fields of the entity using the syntax ``u.name``. Combinations
-of both are also allowed and it is possible to wrap both fields and
-identification values into aggregation and DQL functions. Numerical
-fields can be part of computations using mathematical operations.
-See the sub-section on `Functions, Operators, Aggregates`_ for
-more information.
+The composition of the expressions in the SELECT clause also
+influences the nature of the query result. There are three
+cases:
+
+- All objects. For example:
+    ``SELECT u, p, n FROM Users u...``
+  In this case, the result array will be made up of objects of
+  the type in the FROM clause.  In the example above, the query
+  will return an array of User objects, with associated classes
+  identify else where in the query as 'p' and 'n' hydrated.
```

Comment:
The wording is a bit confusing here - can you rephrase it so that it is clear that `p` and `n` are children of `u`, and are not going to be selected as separate columns in the result?